docs/UserGuide.md

```diff
@@ -6,7 +6,8 @@
 
 # WedLinker User Guide
 
-WedLinker is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+WedLinker is a **desktop app for managing contacts, optimized for use via a Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). </br>
```

Comment:
I think I remember prof saying something like the benefits of GUI being a bit vague, maybe we should expand on this?